<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>favorites2</class>
 <widget class="QDialog" name="favorites2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Favorites</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="FavoritesList">
    <attribute name="title">
     <string>Favorite List</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>741</width>
       <height>301</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>80</number>
     </attribute>
     <column>
      <property name="text">
       <string>Volume ID</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mount Point</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pbAdd">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>390</y>
       <width>181</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVolumeID">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>300</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add A volume</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pbFile">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>300</y>
       <width>41</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbPartition">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>300</y>
       <width>41</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMountPath">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>330</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mount Path</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pbClose">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>390</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMountOptions">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>360</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mount Options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pbFolder">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>300</y>
       <width>41</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ManageVolumeKeys">
    <attribute name="title">
     <string>Manage Keys In Wallets</string>
    </attribute>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>741</width>
       <height>411</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGroupBox" name="groupBoxKeys">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Set Default Wallet</string>
      </property>
      <widget class="QRadioButton" name="rbInternalWallet">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Internal Wallet</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbLibSecret">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Libsecret</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbKWallet">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>KWallet</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbNone">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidgetWallet">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>511</width>
        <height>241</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <column>
       <property name="text">
        <string>Volume Path</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>248</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Volume Path Below</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditVolumePath">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>278</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>308</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Password Below</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>338</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pbAddKeyToWallet">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>370</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbVolumePathFromFavorites">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>278</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>278</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Internal Wallet Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pbChangeWalletPassword">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbShowPassword">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>338</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pbClose_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>370</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Close</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPassword">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>531</width>
        <height>241</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pbVisiblePassword">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>260</y>
        <width>131</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbVolumePathFromFileSystem">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>278</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>pbClose</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>lineEditVolumePath</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>pbAddKeyToWallet</tabstop>
  <tabstop>rbNone</tabstop>
  <tabstop>rbInternalWallet</tabstop>
  <tabstop>rbLibSecret</tabstop>
  <tabstop>rbKWallet</tabstop>
  <tabstop>pbChangeWalletPassword</tabstop>
  <tabstop>tableWidgetWallet</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
